### PR TITLE
Extract fcrepo-camel-webapp to fcrepo4-labs

### PR DESCRIPTION
**Pull request by yinlinchen**
Resolves: https://jira.duraspace.org/browse/FCREPO-1691

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Tomcat or Jetty. It is possible to bundle all of the applications or only select
 modules into the web application.
 
 For more information, see the
-[fcrepo-camel-webapp](https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
+[fcrepo-camel-webapp](https://github.com/fcrepo4-labs/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
```
Comment (acoburn) [inline README.md]: Please update the link to https://github.com/fcrepo4-labs/fcrepo-camel-webapp

File: README.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Then, you can add any combination of the following applications:
 
 If you intend to deploy this application in a web container such as Tomcat or Jetty,
 please refer to the documentation in the
-[fcrepo-camel-webapp](https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
+[fcrepo-camel-webapp](https://github.com/fcrepo4-labs/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
```
Comment (acoburn) [inline README.md]: Please update the link to https://github.com/fcrepo4-labs/fcrepo-camel-webapp

File: fcrepo-fixity/README.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ command from its shell:
 
 If you intend to deploy this application in a web container such as Tomcat or Jetty,
 please refer to the documentation in the
-[fcrepo-camel-webapp](https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
+[fcrepo-camel-webapp](https://github.com/fcrepo4-labs/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
 project.
```
Comment (acoburn) [inline fcrepo-fixity/README.md]: Please update the link to https://github.com/fcrepo4-labs/fcrepo-camel-webapp

File: fcrepo-indexing-solr/README.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ command from its shell:
 
 If you intend to deploy this application in a web container such as Tomcat or Jetty,
 please refer to the documentation in the
-[fcrepo-camel-webapp](https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
+[fcrepo-camel-webapp](https://github.com/fcrepo4-labs/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
```
Comment (acoburn) [inline fcrepo-indexing-solr/README.md]: Please update the link to https://github.com/fcrepo4-labs/fcrepo-camel-webapp

File: fcrepo-indexing-triplestore/README.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ command from its shell:
 
 If you intend to deploy this application in a web container such as Tomcat or Jetty,
 please refer to the documentation in the
-[fcrepo-camel-webapp](https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
+[fcrepo-camel-webapp](https://github.com/fcrepo4-labs/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
```
Comment (acoburn) [inline fcrepo-indexing-triplestore/README.md]: Please update the link to https://github.com/fcrepo4-labs/fcrepo-camel-webapp

File: fcrepo-reindexing/README.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ command from its shell:
 
 If you intend to deploy this application in a web container such as Tomcat or Jetty,
 please refer to the documentation in the
-[fcrepo-camel-webapp](https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
+[fcrepo-camel-webapp](https://github.com/fcrepo4-labs/fcrepo-camel-toolbox/tree/master/fcrepo-camel-webapp)
```
Comment (acoburn) [inline fcrepo-reindexing/README.md]: Please update the link to https://github.com/fcrepo4-labs/fcrepo-camel-webapp

**Comment by acoburn** [comment]
@yinlinchen can you rebase this PR against the master branch?

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/commit/fbe58bd8d52267919b0df4319ab55fe26062fdc1

**Pull request closed without merging**